### DISCUSSION TITLE
Separator with different colors

### DISCUSSION THREAD

**Discussion by thomaschaaf** [twilio-labs/paste#2403] [Ideas] [answered]
Hello,

we used to use material uis `<Divider>` and used the `light` color option for some elements. It would be nice if we could maybe have the option to set the same colors like a [border](https://paste.twilio.design/foundations/colors/#border-colors) or at least `$color-border`, `$color-border-weak` and `$color-border-weaker`?

Thomas

**Comment by TheSisb** [comment, marked as answer]
Hello @thomaschaaf,

You can achieve this functionality today using the [Customization Provider](https://paste.twilio.design/customization/). Here's [an example of how to do it](https://codesandbox.io/s/customizing-separator-whxyxz?file=/src/index.tsx).
